test6/README.md

```diff
@@ -0,0 +1,7 @@
+# Test6
```

Comment:
seems you didn't commit the Git actions files in here

Reply:
Sorry! My answer of test6 was put in .github/workflow. It was included in .gitignore.

<img width="548" alt="image" src="https://user-images.githubusercontent.com/25604384/168715639-44879d23-897a-447e-b430-43173754fbbc.png">

I need to close this PR and resend a new PR.